### ISSUE TITLE
linear regression结果不正确

### ISSUE THREAD

**Issue by jinbooooom** [jinbooooom/OriginDL#6]
iter199: loss = 2.2868562, w = 9.166764, b = 0.34408623
